src/components/homepage/DropdownCart.js

```diff
@@ -0,0 +1,44 @@
+import { Grid } from '@mui/material';
+import React from 'react';
+import CustomSpinner from '../../UI/CustomSpinner';
+import { formatPrice } from '../../ultils';
+
+const DropdownCart = (props) => {
+    const { cart, total, loading } = props;
+
+    const renderCartItem = (arr) => {
+        return arr && arr.map((value, key) => {
```

Comment:
```suggestion
        return arr.map((value, key) => {
```
Nếu đã có loading thì không cần check arr true đâu
Vì khi loading xong thì data (arr) cũng đã trả về và có giá trị rồi